docs/service/processor.rst

```diff
@@ -50,13 +44,42 @@ All helper scripts run in the shell in the container::
 
     $ make shell
 
+Some of the scripts require downloading production data from
+`crash-stats.mozilla.com <https://crash-stats.mozilla.com>`_, and it is
+useful to add an API token before entering the shell.
+
+
+.. _`API token`:
+
+Adding an API Token
+-------------------
+The data download scripts require an API token with the "View Raw Dumps"
+permission to acquire raw crash data. This crash data will not include
```

Comment:
We discovered it requires two different permissions:

* View Raw Dumps
* View Personal Identifiable Information

If you're missing one or the other, then it won't download all the data. So this should be fixed.

Reply:
Do you mean the docs should mention both, or that the API should change, or something else?

Reply:
I think the short-term least-effort solution is that the docs should mention both permissions. Maybe longer term after someone has really looked into it, we should fix the API/permissions structure.